packages/daemon/test/utils/job-runner.test.ts

```diff
@@ -25,6 +26,10 @@ describe('test JobRunner', () => {
     sinon.restore();
   });
 
+  beforeEach(() => {
+    sinon.stub(util, 'copyDir').resolves();
```

Comment:
Not all the cases need to stub `copyDir`, I think we should stub it in the cases not in this hook.

Reply:
just remove this in the hook and add it to the place required